core/src/main/scala/feral/IOSetup.scala

```diff
@@ -20,24 +20,36 @@ import cats.effect.unsafe.IORuntime
 import cats.effect.kernel.Resource
 import cats.effect.IO
 import cats.effect.kernel.Deferred
+import cats.effect.std.Dispatcher
 
 private[feral] trait IOSetup {
+  private def runtime =
+    IORuntime.global
 
-  protected def runtime: IORuntime = IORuntime.global
```

Comment:
We should keep the `runtime` as an overridable `protected def`, in case the user wants to change it.